### DISCUSSION TITLE
Class not found

### DISCUSSION THREAD

**Discussion by chuanshanjia12** [dcm4che/dcm4che#1100] [Q&A]
At org dcm4che3. There is a tag class in the attributes class that cannot be found

**Comment by hczedik** [comment]
The class Tag gets generated by Maven when running `mvn install`.
Please use the "Bug report" template to file a bug. (https://github.com/dcm4che/dcm4che/issues/new?assignees=&labels=&template=bug_report.md&title=)
If this is rather a question, then open a discussion: https://github.com/dcm4che/dcm4che/discussions
